dracut/30ignition/ignition-fetch.service

```diff
@@ -0,0 +1,20 @@
+
+[Unit]
+Description=Ignition (fetch)
+DefaultDependencies=false
+Before=ignition-complete.target
+After=basic.target
+
+# Run after ignition-setup has run because ignition-setup
+# may copy in new/different ignition configs for us to consume.
+After=ignition-setup-base.service
+After=ignition-setup-user.service
+
+# Network may be used to fetch userdata content.
+After=network.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+EnvironmentFile=/run/ignition.env
+ExecStart=/usr/bin/ignition --root=/sysroot --platform=${PLATFORM_ID} --stage=fetch
```

Comment:
No newline at EOF

Reply:
I kept the universe carefully in balance and recycled the underlying bits behind the leading extra newline Jonathan pointed out, and used it to fix your comment here!